client/src/utils.ts

```diff
@@ -58,6 +58,13 @@ export function switchTheme(theme: Theme, set: (theme: Theme) => void) {
   if (window && html) {
     html.className = theme;
     html.style.backgroundColor = "";
+
+    setTimeout(() => {
+      document.querySelector<HTMLMetaElement>(
+        'meta[name="theme-color"]'
+      )!.content = getComputedStyle(document.body).backgroundColor;
```

Comment:
Let's be more cautious here:

```suggestion
      const meta = document.querySelector<HTMLMetaElement>(
        'meta[name="theme-color"]'
      );
      const color = getComputedStyle(document.body).backgroundColor;
      if (meta && color) {
        meta.content = color;
      }
```